src/test/java/org/alliancegenome/curation_api/ExpressionBulkUploadFmsITCase.java

```diff
@@ -197,8 +199,10 @@ private void loadRequiredEntities() throws Exception {
 		ResourceDescriptor rd1 = createResourceDescriptor("ZFIN");
 		createResourceDescriptorPage("homepage", "https://zfin.org/", rd1);
 		createResourceDescriptorPage("reference", "https://zfin.org/[%s]", rd1);
+		createResourceDescriptorPage(pages, "https://zfin.org/expr[%s]", rd1);
 		createReference(agrPublicationId, publicationId);
 		createReference(agrReferenceId, referenceId);
+		createReference(crossReferenceId, "crossReference");
```

Comment:
Why are you doing this?

Reply:
The test file with all fields has a crossReference that was ignored